### PR TITLE
Generate DevTestLab SDK for new version 2018-09-15

### PR DESCRIPTION
**Pull request by jviau**
## Description
Update DevTestLab to 2018-09-15. https://github.com/Azure/azure-rest-api-specs/pull/4357 
---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**
  - being a new API version, does this apply?

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.
  - Have new tests locally, will add in a different PR. This SDK version is already used internally by us, so it has been tested by that.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/DevTestLabs/Management.DevTestLabs/Microsoft.Azure.Management.DevTestLabs.csproj [hunk 1]
```diff
@@ -4,7 +4,7 @@
 	<!-- Please do not move/edit code below this line -->
 
 	<PropertyGroup>
-		<Version>2.0.2</Version>
+		<Version>3.0.0</Version>
 		<Description>Microsoft.Azure.Management.DevTestLabs</Description>
 		<AssemblyName>Microsoft.Azure.Management.DevTestLabs</AssemblyName>
 		<PackageId>Microsoft.Azure.Management.DevTestLabs</PackageId>
```
Comment (dsgouda) [inline src/SDKs/DevTestLabs/Management.DevTestLabs/Microsoft.Azure.Management.DevTestLabs.csproj]: Please update `PackageReleaseNotes`

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please fix the failing tests

**Comment by jviau** [comment]
@dsgouda I have fixed the test and updated release notes.

**Comment by jviau** [comment]
ping @dsgouda. Any other changes I need to make?

**Review by dsgouda — APPROVED** [review]
LGTM

**Comment by jviau** [comment]
Thank you @dsgouda. Do you know when the nuget package will be published? Or is there more steps before it is published.

**Comment by dsgouda** [comment]
@jviau Steps to publish could be found [here](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md)

**Pull request merged**